### PR TITLE
feat: Updated Navbar to be uniform with umaproject.org

### PR DESCRIPTION
**Pull request by Tulun**
**Motivation**
Navbar should be aligned with our marketing website.

**Summary**

Header updated on mobile and desktop.

Note:

MenuToggle.tsx and MobileMenu.tsx we're basically just pulled out from the old Header.tsx file.

Screen Shot 2021-08-12 at 10 08 40 AM
Screen Shot 2021-08-12 at 10 08 51 AM

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [x]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

**Comment by daywiss** [comment]
just want to mention this quickly, the search bar does not seem to be working, cant type in it

**Comment by Tulun** [comment]
> just want to mention this quickly, the search bar does not seem to be working, cant type in it

Addressed this strange, strange bug via removing position: sticky.

File: hooks/useUserClickedOutsideElement.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+export default function useUserClickedOutsideElement(
+  ref: RefObject<HTMLDivElement>,
+  callback: () => void
+) {
+  useEffect(() => {
+    /**
+     * Alert if clicked on outside of element
+     */
+    function handleClickOutside(event: Event) {
+      if (ref && ref.current && !ref.current.contains(event.target as Node)) {
```
Comment (daywiss) [inline hooks/useUserClickedOutsideElement.ts:15]: interesting trick

**Review by daywiss — APPROVED** [review]
works well. id just suggest you comment in the pr on which files did/didnt change, since theres also a refactor in here. probably a lot less code than it seems actually does this

**Pull request merged**